com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/ShaderPass.cs.hlsl

```diff
@@ -31,8 +31,9 @@
 #define SHADERPASS_RAYTRACING_GBUFFER (21)
 #define SHADERPASS_RAYTRACING_SUB_SURFACE (22)
 #define SHADERPASS_PATH_TRACING (23)
-#define SHADERPASS_CONSTANT (24)
-#define SHADERPASS_FULL_SCREEN_DEBUG (25)
+#define SHADERPASS_RAY_TRACING_DEBUG (24)
```

Comment:
Does this define work? Elsewhere you're using SHADERPASS_RAYTRACING_DEBUG (without underscore in the middle).

Reply:
It's automatically generated, (RayTracing => RAY_TRACING, raytracing => RAYTRACING). Should be with a T upper case